packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -278,6 +285,9 @@ class DebuggerController extends DisposableController
   /// Return the sorted list of ScriptRefs active in the current isolate.
   ValueListenable<List<ScriptRef>> get sortedScripts => _sortedScripts;
 
+  bool get centerScrollLocation => _centerScrollLocation;
```

Comment:
is it possible that this value could get accessed for a script it is not associated with? Storing this at the controller level vs the per call level seems problematic